DependencyInjection/DoctrineExtension.php

```diff
@@ -761,6 +763,21 @@ private function loadPropertyInfoExtractor($entityManagerName, ContainerBuilder
         $propertyExtractorDefinition->addTag('property_info.type_extractor', ['priority' => -999]);
     }
 
+    /**
+     * Loads a validator loader for each defined entity manager.
+     */
+    private function loadValidatorLoader(string $entityManagerName, ContainerBuilder $container) : void
+    {
+        if (! class_exists(DoctrineLoader::class)) {
```

Comment:
This requires testing for LoaderInterface first as the bridge does not prevent installing its latest version without the Validator component (which is expected) and then that would be a fatal error due to the missing interface.